pkg/std/package_list.go

```diff
@@ -47,6 +47,7 @@ var StdPackages = map[string]struct{}{
 	"debug/macho":                          {},
 	"debug/pe":                             {},
 	"debug/plan9obj":                       {},
+	"embed":                                {},
```

Comment:
this file is autogenerated and after regeneration all changed stuff will be missed